tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHelpers.cs

```diff
@@ -170,8 +170,13 @@ public static string GetMsiImdsErrorResponse()
                 "\"correlation_id\":\"77145480-bc5a-4ebe-ae4d-e4a8b7d727cf\",\"error_uri\":\"https://westus2.login.microsoft.com/error?code=500011\"}";
         }
 
-        public static string CreateClientInfo(string uid = TestConstants.Uid, string utid = TestConstants.Utid)
+        public static string CreateClientInfo(string uid = TestConstants.Uid, string utid = TestConstants.Utid, bool CreateClientInfoForS2S = false)
         {
+            if (CreateClientInfoForS2S)
+            {
+                return Base64UrlHelpers.Encode("{\"xms_acb\":[\"value1\",\"value2\"]}");

```

Comment:
What is `xms_acb`? Can you add some comment to explain this?

Reply:
Actually, MSAL code should not mention it because it should treat these values as generic